test/release/rollup-es.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+alias uglify-js=$PWD/bin/uglifyjs
+rm -rf tmp/rollup
+git clone https://github.com/rollup/rollup tmp/rollup
+cd tmp/rollup
+rm -rf .git/hooks
+git checkout 3d80c06f895eab41e648ee99786fa68c72458b80
+patch -l -p1 <<EOF
+--- a/rollup.config.js
++++ b/rollup.config.js
+@@ -1,5 +1,4 @@
+ import { readFileSync } from 'fs';
+-import buble from 'rollup-plugin-buble';
+ import resolve from 'rollup-plugin-node-resolve';
+ import commonjs from 'rollup-plugin-commonjs';
+ import json from 'rollup-plugin-json';
+@@ -25,12 +24,6 @@ export default [
+         input: 'src/node-entry.js',
+         plugins: [
+             json(),
+-            buble({
+-                include: ['src/**', 'node_modules/acorn/**'],
+-                target: {
+-                    node: '4'
+-                }
+-            }),
+             resolve(),
+             commonjs()
+         ],
+@@ -48,12 +41,6 @@ export default [
+         input: 'src/browser-entry.js',
+         plugins: [
+             json(),
+-            buble({
+-                include: ['src/**', 'node_modules/acorn/**'],
+-                target: {
+-                    node: '4'
+-                }
+-            }),
+             resolve(),
+             commonjs(),
+             {
+@@ -80,7 +67,6 @@ export default [
+         plugins: [
+             string({ include: '**/*.md' }),
+             json(),
+-            buble({ target: { node: 4 } }),
+             commonjs({
+                 include: 'node_modules/**'
+             }),
+--- a/test/mocha.opts
++++ b/test/mocha.opts
+@@ -1,2 +1 @@
+---require buble/register
+ test/test.js
+EOF
+echo '> uglify-js' $@
+for i in `find src -name '*.js'`
+do
+    echo "$i"
+    cp -f "$i" "$i.bak"
+    uglify-js "$i.bak" $@ > "$i" && rm -f "$i.bak"
+done
+rm -rf node_modules
+npm ci
+npm test
```

Comment:
Instead of `npm test` here, for extra coverage we could run `npm run build`, then uglify the dist/ js files in-situ and then run `node_modules/.bin/mocha` - similar to butternut.sh above.

Reply:
Done in https://github.com/mishoo/UglifyJS/pull/4741/commits/133c476a33a81c412c34198c6057aa03fca80124